File: ruby/csv/sample.rb
#! /usr/bin/ruby
# -*- coding: utf-8 -*-

require "csv";
count = 0

## あまり速くない
CSV.open("/home/kzk/dataset/drbd/73789/contents_1000.tsv", mode="r", fs="\t") do |line|
  print(count, ": \n")
  #puts(line)
  count += 1
end

File: python/string/sample.py
#!/usr/bin/python
# -*- coding: utf-8 -*-

## 文字列中で式展開をする

# 文字列 in 文字列
instr = "test"
print "%(instr)s" % locals()

# 数値 in 文字列
num = 10
print "%(num)d" % locals()


